src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+import CustomFieldInputMultipleChoice from './custom-field-input-multiple-choice.jsx';
+
+describe('<CustomFieldInputMultipleChoice>', () => {
```

Comment:
This is usually the path to the component--in this case `components/custom-field-input-multiple-choice/custom-field-input-multiple-choice`. Is that what we want to have instead?

Reply:
I just checked and it seems we're all over the place in our MDS repo. Hmmm... 

I think we can do either, as long as we're reasonable. 

- Use `<ComponentName>` to indicate that the test suite is for a component
- Use `ComponentName component` to indicate the test suite is for a component
- Use `src/components/component-name` to indicate the test suite is for a component

In contrast to linters:

- Use `ruleName linter` to indicate that the test suite is for a linter
- Use `src/linter/rule-name` to indicate the test suite is for a linter